src/cli/func-api.ts

```diff
@@ -16,6 +17,44 @@ export class FuncAPI {
     return funcCliCommand(['version']);
   }
 
+  static invokeFunctionLocal(
+    id: string,
+    location: string,
+    contextType: string,
+    format: string,
+    source: string,
+    type: string,
+    data: string,
+  ): string {
+    if (id) {
+      return `func invoke --id ${id} -p ${location} --content-type ${contextType} -f ${format} --source ${source} --type ${type} --data ${data}`;
+    }
+    return `func invoke -p ${location} --content-type ${contextType} -f ${format} --source ${source} --type ${type} --data ${data}`;
+  }
+
+  static invokeFunctionRemote(
+    id: string,
+    namespacs: string,
```

Comment:
This should be `namespace` and changed across the files.

Reply:
done